File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if (nums.length === 0 || nums.length === 1){
        return nums.length;
    }
    var left = 0 
    for (var right = 0; right < nums.length; right ++){
        if (nums[right] !== nums [right+1]){
            nums[left++] = nums[right]
        }
    }
    return left
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l,r = 0, 1 
        while r < len(nums):
            if nums[r] != nums[l]:
                nums[l+1] = nums[r]
                l += 1
            r += 1
        return len(nums[:l+1])